lib/plugins/invoke.js

```diff
@@ -50,6 +50,13 @@ class Invoke {
           raw: {
             usage: 'Flag to pass input data as a raw string',
           },
+          context: {
+            usage: 'Context of the service',
+          },
+          contextPath: {
+            usage: 'Path to JSON or YAML file holding context data',
+            shortcut: 'x',
```

Comment:
Let's remove that shortcut

(I've just realized we've embraced them originally on `invoke local`, but they are error prone, and I believe it's better to remove them from `invoke local` at some point, than embracing same pattern here)